public/script.js

```diff
@@ -131,22 +131,78 @@ function SidebarToggle() {
   const navbar = document.querySelector(".left");
   if (!navbar.classList.contains('menu-closed') && navbar.classList.contains('open-menu')) {
     closeSidebar();
-  } 
+  }
   else {
     openSidebar();
     document.body.addEventListener('click', function(event) {
       var sidebar = document.getElementsByClassName('left')[0]; //Ensuring that clicks inside the sidebar(outside lobbies) don't make the sidebar collapse
       var lobbyName = document.getElementsByClassName('person'); //In case user clicks a lobby from the sidebar, the sidebar collapse. This is a list of classes
       var addLobbyPage = document.getElementById('room'); //Ensuring that clicks on page to add lobby names doesn't make the sidebar collapse
-      for (var i=0; i<lobbyName.length; i++) { 
+      for (var i=0; i<lobbyName.length; i++) {
         if(lobbyName[i].contains(event.target)) {
-          closeSidebar(); 
+          closeSidebar();
           return;
         }
       }
-      if(!sidebar.contains(event.target) && !addLobbyPage.contains(event.target)) { 
-        closeSidebar(); 
+      if(!sidebar.contains(event.target) && !addLobbyPage.contains(event.target)) {
+        closeSidebar();
       }
-    }, false); 
+    }, false);
   }
 }
+
+function processMsg(raw_msg) {
+
```

Comment:
Why is this function not in client.js from where is called?

Reply:
I added this function there(script.js), let me resolve the conflicts, then It will be fine.

Reply:
I mean that shouldn't this function be in client.js? Since it's being called from there.

P.S. better way to fix conflicts with master is to use git rebase master instead of merge, this doesn't create "merge" commit and you have clean commit history.

Reply:
Cool let's keep it here then.